src/main/java/com/ing/data/cassandra/jdbc/CassandraResultSet.java

```diff
@@ -886,6 +886,10 @@ public Object getObject(final int columnIndex) throws SQLException {
         final DataType cqlDataType = getCqlDataType(columnIndex);
         final DataTypeEnum dataType = fromDataType(cqlDataType);
 
+        if (currentRow.isNull(columnIndex - 1)) {
```

Comment:
Good point @kornilova203!
So, the same check should be done in the method `getObject(String)`:
https://github.com/ing-bank/cassandra-jdbc-wrapper/blob/35e47fecfdde045ae99223df86beef7f6f83075f/src/main/java/com/ing/data/cassandra/jdbc/CassandraResultSet.java#L1020-L1024

Reply:
Yes, you're right. I keep this for a future refactoring.